index.html

```diff
@@ -1,29 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css">
-        <link rel="stylesheet" href="css/index.css">
-        <title>Galite Framework</title>
-    </head>
-    <body>
-        <div id="content">
-            <img src= "images/logo_galite.png" id="galite-logo"/>
-            <center>
-                <div id= "container">
-                    <p id="title"> <b>Welcome to Galite Framework</b></p>
-                    <p id="description">A framework for accelerating building business applications</p>
-                    <ul id="list-inline">
-                        <li class="list-inline-item"> <a class="link" href= "user-guide.html">  <i class="fas fa-book"></i>&nbsp; User Guide</a> </li>
-                        <li class="list-inline-item"> <a class="link" href= "https://maven-badges.herokuapp.com/maven-central/org.kopi/galite-core"> <i class="fas fa-download"></i>&nbsp; v1.0.0-beta</a> </li>
-                        <li class="list-inline-item"> <a class="link" href= "https://github.com/kopiLeft/Galite"> <i class="fab fa-github"></i>&nbsp; Code & Issues</a> </li>
-                        <li class="list-inline-item"> <a class="link" href= "user-guide.html#_getting_started"> <i class="fa fa-power-off"></i>&nbsp; Getting Started</a> </li>
-                    </ul>
-                </div>
-            </center>
-        <div>
-            <h1 id="section">About</h1>
-            <p id="text"> Galite is an opensource framework of kopiLeft that allows users to build business applications with forms, reports and charts using a simple, creative and specific syntax that will allow you to put all the verbosity aside and concentrate on the business logic with its simple domain specific language.</p>
-        </div>
-        </div>
-    </body>
-</html>
+
+<head>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css" />
+  <link rel="stylesheet" href="css/index.css" />
+  <title>Galite Framework</title>
+</head>
+
+<body>
+  <main id="main">
+    <img src="images/logo_galite.png" id="galite-logo" class="main-logo" />
+    <h3 class="main-paragraph">
+
+      an opensource framework that allows users to build business applications with forms, reports and charts using a
+      simple, creative and specific syntax based on a simple domain specific language.
+    </h3>
+    <div class="main-actions">
+      <a class="btn btn--primary" href="user-guide.html#_getting_started">
+        Get started
+      </a>
+      <a href="https://github.com/kopiLeft/Galite" class="btn btn-outline--secondary">
+        <i class="fab fa-github"></i>&nbsp; Github
+      </a>
```

Comment:
can you add new button for maven release : https://search.maven.org/artifact/org.kopi/galite-core/1.0.0-beta/jar